000-vip-init.php

```diff
@@ -233,6 +233,15 @@
 require_once __DIR__ . '/vip-helpers/class-user-cleanup.php';
 require_once __DIR__ . '/vip-helpers/class-wpcomvip-restrictions.php';
 
+// Load the Telemetry files
+require_once __DIR__ . '/telemetry/class-telemetry-system.php';
+require_once __DIR__ . '/telemetry/class-tracks.php';
+require_once __DIR__ . '/telemetry/tracks/class-tracks-event-dto.php';
+require_once __DIR__ . '/telemetry/tracks/class-tracks-event.php';
+require_once __DIR__ . '/telemetry/tracks/class-tracks-event-queue.php';
+require_once __DIR__ . '/telemetry/tracks/class-tracks-client.php';
```

Comment:
This will likely lead to intermittent fatals on the initial deploy as this file will be updated sooner than the subdirectories.

Reply:
How would you like this part changed? Would a follow-up PR with these lines work?

Reply:
Please let me know if this approach works:  https://github.com/Automattic/vip-go-mu-plugins/pull/5869/commits/c95a993f6199ac614d11c1089ed49203db95969e

Reply:
I'm not sure if this works. Are there any cross-dependencies? I.e. where telemetry/class-tracks.php trying to include `telemetry/tracks/class-tracks-client.php`? I.e. the situation is - the file A was already copied and is passing file_exists check. The file A wants something from file C, which is not copied yet.

Reply:
These files do not have side effects, only class definitions, so it should not be an issue.
And neither of these files includes or requires other files directly.

Reply:
I changed this part to only require files if all of them are present. So if there's a missing file, none will be required.